data/mods/Xedra_Evolved/recipes/alchemy.json

```
@@ -192,6 +192,20 @@
     "tools": [ [ [ "surface_heat", 25, "LIST" ] ] ],
     "components": [ [ [ "faewild", 1 ] ], [ [ "scrap_dreamdross", 2 ] ], [ [ "bone", 1 ] ], [ [ "corpse_ash", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "nether_snare",
+    "category": "CC_XEDRA",
+    "subcategory": "CSC_XEDRA_ALCHEMY",
+    "skill_used": "fabrication",
+    "skills_required": [ "deduction", 1 ],
+    "difficulty": 3,
+    "time": "75 m",
+    "flags": [ "SECRET" ],
+    "tools": [ [ [ "paint_brush", -1 ] ] ],
+    "components": [ [ [ "paint", 1, "LIST" ] ], [ [ "scrap_dreamdross", 3 ] ], [ [ "mutant_blood", 1 ] ], [ [ "silver_small", 1 ] ] ]
```

```suggestion
    "tools": [ [ [ "paint_brush", -1 ] ], [ [ "paint", 1, "LIST" ] ] ],
    "components": [ [ [ "scrap_dreamdross", 3 ] ], [ [ "mutant_blood", 1 ] ], [ [ "silver_small", 1 ] ] ]
```

Resolves the load error and makes the recipe actually craftable